docs/guide/what-to-do-if-you-have-been-hacked.md

```diff
@@ -0,0 +1,16 @@
+# What to do if you have been hacked
+
+As a first step if you have any suspicion that your device's security may have been compromised move the contents of your wallet to a 
+new safe wallet as soon as possible and cease using it. Once this has been done please contact one of the team, and we will be able to offer advice
+on a case by case basis. Unfortunately, due to the nature of blockchain there are considerable limitations on the help that we 
+are able to provide to those who have been hacked. Here is a few of the things we are and are not able to offer:
+
+* WE CANNOT transfer any of the tokens minted from the hacked account to another one
+* WE CANNOT divert any funds that have been sent to that account
+* WE CAN help you create a new artist account
+* WE CAN change the name of the compromised profile to indicate that this is a hacked/inactive account
+* WE CAN stop the hacked account from being able to mint any new tokens
+
```

Comment:
All points are good - it feels more like `what we can do` list than `what to do when hacked` FAQ.

I am wonder if we can list the process of what if compromised.

1. Reach out to the team and let them know
2. They will confirm its your over your verify socials
3. Create a new account on KO
4. Provide this to the KO team and indicate old and new and the reason
5. KO team will disable minting of the old and enable the new
6. KO team will re-name your old profile to indicate to anyt collectors that this accout is no longer yourself

Also maybe a note in a message box:

```
::: warning

We are a non-custoidal platform so migrations of blockchain history or ownership of token is limited by design,  we will do all we can to help but onchain ownerhsip is out of our control
:::
```

Or something similar to this?